### DISCUSSION TITLE
How to see disk space left on e-reader in koreader?

### DISCUSSION THREAD

**Discussion by fritzrehde** [koreader/koreader#11130] [Q&A]
Is there a way to display the available space on the e-reader? I would like to know before I run out of storage. Thanks!

**Comment by Galunid** [comment]
`Top Menu -> Last Tab -> System statistics` it's under `Storage information`

**Reply by fritzrehde** [reply]
Thanks! Is it possible to display this information in the status bar?

**Reply by Galunid** [reply]
I don't think so, you could open Feature Request ticket if you want

**Reply by Frenzie** [reply]
That could be surprisingly complex. You'd need to select one or more to care about. But I suppose you could always start with just the home directory... still, I predict someone will ask for more very swiftly. E.g., I stick most of my stuff on my 64 GB microSD, not the 4GB internal one.

**Reply by NiLuJe** [reply]
That codepath is also extremely not pretty, so you can bet your ass I'd veto it for the footer anyway ;).
